src/utils.ts

```diff
@@ -56,16 +56,26 @@ export function serial<T>(
   );
 }
 
+type Task = ReturnType<typeof console.createTask>;
+const defaultTask = { run: (function_) => function_() } as Task;
+const createTask =
+  typeof console.createTask !== "undefined"
+    ? console.createTask
+    : () => defaultTask;
```

Comment:
Refactor: Maybe we can create a `_createTask` function that is typed as `console.createTask`? (Also for future refactors, we might actually somehow respect the `name` in shim implementation.

Reply:
Makese sense (i'm basically suggesting to move L64 that returns default task into it's own variable and type better.